### PR TITLE
Typo - Issue #104

### PR DESCRIPTION
**Pull request by GuillaumeHullin**
(sorry, I thought I included it in yesterday PR)

### CODE REVIEW

**Review by rbicelli — CHANGES_REQUESTED** [review]
Please leave the top lines with buy me a coffeee badge as they are.

File: README.md [hunk 1]
```diff
@@ -36,25 +36,78 @@ Tested with pfSense 2.5.x, Zabbix 4.0, Zabbix 5.0
...
 ## Configuration
 
-First copy the file pfsense_zbx.php to your pfsense box (e.g. to /root/scripts).
+### Install PHP script for Agent
 
-From **Diagnostics/Command Prompt** input this one-liner:
+- Option 1: via Web GUI **Diagnostics/Command Prompt**
 
 ```bash
-curl --create-dirs -o /root/scripts/pfsense_zbx.php https://raw.githubusercontent.com/rbicelli/pfsense-zabbix-template/master/pfsense_zbx.php
```
Comment (rbicelli) [inline README.md:47]: Please leave my repository as default

**Comment by GuillaumeHullin** [comment]
Sorry, I went a bit quick on the copy/paste from my repo

File: template_pfsense_active_ipsec.xml [hunk 2]
```diff
@@ -4,20 +4,20 @@
...
-            <name>pfSense Active: IPsec</name>
+            <name>Template Futur-Tech App pfSense Active: IPsec</name>
             <description>Extension for pfSense Active Template.&#13;
 Monitor IPsec VPN.&#13;
 &#13;
 Requires pfsense_zbx.php installed to pfSense Box.&#13;
 Version 1.0.2&#13;
 &#13;
-https://github.com/rbicelli/pfsense-zabbix-template</description>
+https://github.com/Futur-Tech/futur-tech-zabbix-pfsense</description>
```
Comment (rbicelli) [inline template_pfsense_active_ipsec.xml:20]: Please leave reference to original repository.
You can maintain your own branch but for merging to main repo it is mandatory to leave links pointing there

File: template_pfsense_active_ovpn_user.xml [hunk 3]
```diff
@@ -4,23 +4,23 @@
     <date>2021-07-12T10:37:03Z</date>
     <groups>
         <group>
-            <name>Templates/Network Devices</name>
+            <name>Templates/Futur-Tech/Applications</name>
```
Comment (rbicelli) [inline template_pfsense_active_ovpn_user.xml:7]: Please leave reference to original category.

File: template_pfsense_active_ovpn_user.xml [hunk 4]
```diff
@@ -4,23 +4,23 @@
...
         </group>
     </groups>
     <templates>
         <template>
             <template>Template pfSense Active OpenVPN Server User Auth</template>
-            <name>pfSense Active: OpenVPN Server User Auth</name>
+            <name>Template Futur-Tech App pfSense Active OpenVPN Server User Auth</name>
```
Comment (rbicelli) [inline template_pfsense_active_ovpn_user.xml:13]: Please leave reference to original category.

File: template_pfsense_active_ovpn_user.xml [hunk 5]
```diff
@@ -4,23 +4,23 @@
...
             <description>Extension for pfSense Active Template.&#13;
 Monitor client Connections of OpenVPN Server.&#13;
 &#13;
 Requires pfsense_zbx.php installed to pfSense Box.&#13;
 Version 1.0.2&#13;
 &#13;
-https://github.com/rbicelli/pfsense-zabbix-template</description>
+https://github.com/Futur-Tech/futur-tech-zabbix-pfsense</description>
```
Comment (rbicelli) [inline template_pfsense_active_ovpn_user.xml:20]: Please leave reference to original maintainer.

File: template_pfsense_active_ovpn_user.xml [hunk 6]
```diff
@@ -4,23 +4,23 @@
...
             <groups>
                 <group>
-                    <name>Templates/Network Devices</name>
+                    <name>Templates/Futur-Tech/Applications</name>
```
Comment (rbicelli) [inline template_pfsense_active_ovpn_user.xml:23]: Please leave reference to original category.

File: template_pfsense_active_speedtest.xml [hunk 7]
```diff
@@ -1,26 +1,26 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <zabbix_export>
     <version>5.0</version>
-    <date>2021-07-12T10:37:45Z</date>
+    <date>2021-07-16T08:36:57Z</date>
     <groups>
         <group>
-            <name>Templates/Network Devices</name>
+            <name>Templates/Futur-Tech/Modules pfSense</name>
```
Comment (rbicelli) [inline template_pfsense_active_speedtest.xml:7]: Please leave reference to original category.

File: template_pfsense_active_speedtest.xml [hunk 8]
```diff
@@ -1,26 +1,26 @@
...
         </group>
     </groups>
     <templates>
         <template>
             <template>pfSense Active Speedtest</template>
-            <name>pfSense Active: Speedtest</name>
+            <name>Template Futur-Tech App pfSense Active Speedtest</name>
```
Comment (rbicelli) [inline template_pfsense_active_speedtest.xml:13]: Please leave reference to original category.

File: template_pfsense_active_speedtest.xml [hunk 9]
```diff
@@ -1,26 +1,26 @@
...
             <description>Extension for pfSense Active Template.&#13;
 Executes Speedtests on WAN Interfaces.&#13;
 &#13;
 Requires pfsense_zbx.php installed to pfSense Box.&#13;
 Version 1.0.5&#13;
 &#13;
-https://github.com/rbicelli/pfsense-zabbix-template</description>
+https://github.com/Futur-Tech/futur-tech-zabbix-pfsense</description>
```
Comment (rbicelli) [inline template_pfsense_active_speedtest.xml:20]: Please leave reference to original maintainer.

File: template_pfsense_active_speedtest.xml [hunk 10]
```diff
@@ -1,26 +1,26 @@
...
             <groups>
                 <group>
-                    <name>Templates/Network Devices</name>
+                    <name>Templates/Futur-Tech/Modules pfSense</name>
```
Comment (rbicelli) [inline template_pfsense_active_speedtest.xml:23]: Please leave reference to original category.

**Review by rbicelli — COMMENTED** [review]
The main thing is if you want to merge to the main repo you need to keep the original references and not to rebrand the template.

**Comment by GuillaumeHullin** [comment]
Yes, I fully understand.
I rebranded in my repo in order to fit with my other repos and my Zabbix setup.

I tried to make a middle forked-repo between yours and my main to avoid  pushing my branding changes but I clearly mixed up myself. 

Really sorry for the inconvenience. 

I'm gonna review everything and ask for review once it's done.

**Comment by GuillaumeHullin** [comment]
I think I found why my PR changed more than what I planned on. When I created this PR almost a year ago, I should have created a separate branche in my fork... because all the changes with Futur-Tech branding that cames months after changed the content of this PR.
Lesson learned.

**Review by rbicelli — APPROVED** [review]

**Pull request merged**